tests/Util/Fixtures/TraversableSerializableClass.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tests\Util\Fixtures;
+
+class TraversableSerializableClass implements \IteratorAggregate
```

Comment:
Do we really need this custom class? Can't we use directly `ArrayIterator`?

Reply:
Actually yes, you are right